publishing.gradle

```diff
@@ -0,0 +1,165 @@
+// this relies on a vault connection or the presence of a publish.properties file with:
+// bintray_api_key
+// bintray_username
+
+
+def gitUrl = "https://github.com/uport-project/uport-android-sdk.git"
+def groupID = "com.github.uport-project.uport-android-sdk"
+def siteUrl = "https://developer.uport.me"
+def sdkDescription = "uPort SDK for android"
+
+subprojects { subproject ->
+
+    afterEvaluate {
+
+        if (subproject.plugins.hasPlugin('com.github.dcendents.android-maven')) {
+
+            subproject.group = groupID
+            subproject.version = uport_sdk_version
+            subproject.description = subproject.ext.has("description") ? subproject.ext.description : sdkDescription
+
+            //pom fields required by maven central
+            install {
+                repositories.mavenInstaller {
+                    pom.project {
+                        packaging 'aar'
+                        groupId subproject.group
+                        artifactId subproject.name
+
+                        name subproject.name
+                        description subproject.description
+
+                        url siteUrl
+
+                        licenses {
+                            license {
+                                name "The Apache Software License, Version 2.0"
+                                url "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                            }
+                        }
+                        developers {
+                            developer {
```

Comment:
I only added myself because this requires an email address.

Reply:
ok